os/linux/iahwc.h

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OS_LINUX_IAHWC_H_
+#define OS_LINUX_IAHWC_H_
```

Comment:
Is it ok to keep this as OS_LINUX_IAHWC_H? Because although this header resides in os/linux, it is exported to ${include}/libiahwc/ along with the headers in public/

Reply:
k